src/blacklist.js

```diff
@@ -0,0 +1,12 @@
+const Url = require("url");
```

Comment:
Don't rename the module name as if it was a class. Call it `url` here and rename the argument in the `isInBlacklist` function to `uri` or `url_`.